src/parser/tptp/Tptp.g

```diff
@@ -698,17 +698,17 @@ definedFun[CVC4::ParseOp& p]
                   MK_TERM(api::GT, decPart, SOLVER->mkReal(1, 2)),
                   // if decPart > 0.5, round up
                   MK_TERM(api::TO_INTEGER,
-                          MK_TERM(api::PLUS, n, SOLVER->mkReal(1))),
+                          MK_TERM(api::PLUS, n, SOLVER->mkInteger(1))),
                   // if decPart == 0.5, round to nearest even integer:
                   // result is: to_int(n/2 + .5) * 2
                   MK_TERM(api::MULT,
                           MK_TERM(api::TO_INTEGER,
                                   MK_TERM(api::PLUS,
                                           MK_TERM(api::DIVISION,
                                                   n,
-                                                  SOLVER->mkReal(2)),
+                                                  SOLVER->mkInteger(2)),
```

Comment:
Is it necessary that this change happens?

Reply:
No, but I think it is better to have `Rational(2)` here than `(TO_REAL Rational(2))`

Reply:
So what @mudathirmahgoub is right: after this PR, it is more *efficient* to use mkInteger here since it avoids a cast.  However, it is less intuitive, since this means that we are assuming that DIVISION is polymorphic, allowing Real divided by Int.

Alternatively we could just multiply by 0.5 here :-P

Reply:
I'd say efficiency is of no consequence for this so I'd go for what is more clear.

Reply:
Done by going for what is more clear